lib/ball.js

```diff
@@ -48,9 +45,9 @@ var obstacleCoords = [[50, 50, 250, 60], [50, 450, 250, 460], [50, 50, 60, 450],
 
 Ball.prototype.holeCheck = function(puttHole) {
   if(Math.abs((puttHole.x - this.x)) < (puttHole.radius + this.radius) && Math.abs((puttHole.y - this.y)) < (puttHole.radius + this.radius)) {
-      this.x = puttHole.x
-      this.y = puttHole.y
-      this.moving = false
+    this.x = puttHole.x
+    this.y = puttHole.y
+    this.moving = false
```

Comment:
Im going to add a inHole attribute so we know when to switch levels and stop stroke count